swagger.yaml

```diff
@@ -1384,37 +1142,25 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
-  /clusters/{cluster_id}/hosts:
+
+  /infra-envs:
     post:
       tags:
         - installer
-      security:
```

Comment:
no security for post?

Reply:
It has agentAuth (users don't call RegisterHost)